src/app/organization/organization-routing.module.ts

```diff
@@ -110,11 +111,25 @@ const routes: Routes = [
             },
             {
               path: ':id',
-              component: ViewEmployeeComponent,
-              data: { title: extract('View Employee'), routeResolveBreadcrumb: ['employee', 'displayName'] },
-              resolve: {
-                employee: EmployeeResolver
-              }
+              data: { title: extract('View Employee'), routeParamBreadcrumb: 'id' },
+              children: [
+                {
+                  path: '',
+                  component: ViewEmployeeComponent,
+                  resolve: {
+                    employee: EmployeeResolver,
+                  }
+                },
+                {
```

Comment:
@spider0061 dont use offices resolver, create a new edit employee resolver in edit employee component.ts folder, take reference from #664 , make a new api request for employee and set template = true as http param, this will bring a field of allowed offices use that as office data, you can see api request in community app page.

Reply:
make this api request go through edit employee resolver